tests/phpunit/tests/test-option-schema.php

```diff
@@ -0,0 +1,481 @@
+<?php
+
+use WP_Syntex\Polylang\Options\Business;
+
+/**
+ * Test the schema of all classes extending {@see WP_Syntex\Polylang\Options\Abstract_Option}.
+ */
+class Option_Schema_Test extends WP_UnitTestCase {
```

Comment:
Since there is no factory used, no set up and all. Is `WP_UnitTestCase` useful?
Could we just use `PHPUnit_Adapter_TestCase`?
Sorry to think about it only now... It's more a generic question than a change request.

Reply:
Done in [17a1da4](https://github.com/polylang/polylang/pull/1485/commits/17a1da42b091dd0a1844d070d9c027a83f19315d).